app/src/test/java/org/mozilla/fenix/components/appstate/AppStoreReducerTest.kt

```diff
@@ -0,0 +1,30 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.components.appstate
+
+import org.junit.Assert.assertEquals
+import org.junit.Test
+import org.mozilla.fenix.wallpapers.Wallpaper
+
+class AppStoreReducerTest {
```

Comment:
I don't know if we typically have tests against the reducer (vs the Store). Normally, I would expect that we write tests against the AppStore and dispatch the actions and check its resulting state.